parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -88,7 +88,7 @@
       NameMapping nameMappingFromExpectedSchema = MappingUtil.create(expectedSchema);
       fileSchema =
           ParquetSchemaUtil.applyNameMapping(
-              RemoveIds.removeIds(fileSchema), nameMappingFromExpectedSchema);
+              RemoveIds.removeIds(fileSchema), nameMappingFromExpectedSchema, true);
```

Comment:
Can we use something like https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java#L901 to avoid changes to `ParquetUtil` and `ParquetSchemaUtil`?

Reply:
Use `ApplyNameMappingCaseInsensitive` instead.